migrations/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
What is the purpose of this readme?

Reply:
I pushed the Migrations folder for it to work in Heroku which came with this README

Reply:
ok, please add a new line here.

Reply:
Done